packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -29,22 +25,32 @@ export type SynapseChatProps = {
 type ChatInteraction = {
   userMessage: string
   chatResponseText?: string
+  chatErrorReason?: string
```

Comment:
Keep track of any error when attempting to create a new chat interaction (and surface it in the UI).